client/app/components/packages/pas-form/show.hbs

```diff
@@ -13,18 +12,63 @@
       </p>
     </section>
 
-    <section class="form-section" id="#applicant-team">
+    <section class="form-section" id="applicant-team">
       <h3>Applicant Team</h3>
+      {{#each @package.pasForm.applicants as |applicant|}}
+        <h4>
+          {{applicant.dcpFirstname}} {{applicant.dcpLastname}} 
+          <small>({{applicant.targetEntity}})</small>
+        </h4>
 
-      {{!-- TODO: List team members --}}
-      <div class="bg-red-dim xlarge-padding medium-margin-bottom">List team members</div>
+        {{#if applicant.dcpOrganization}}
+        <p>
+          <strong>
+            Organization
+          </strong>
+          <br>
+          {{applicant.dcpOrganization}}
+        </p>
```

Comment:
I meaaaaan, I think this should be indented but just a nit. Wish our template linter could catch that kinda stuff.

Reply:
Ahh yea was hoping linter would have caught it, good eye but we were just getting this in rough and dirty to be able to go back when we have a strategy for broader component architecture.